cubedash/generate.py

```diff
@@ -131,7 +131,7 @@ def print_status(product_name=None, year=None, month=None, day=None, summary=Non
         log.warning("product.unsupported", reason=e.reason)
         return product_name, GenerateResult.UNSUPPORTED, None
     except Exception:
-        log.exception("product.error", exc_info=True)
+        log.exception("product.error")
```

Comment:
I have a feeling this is a false alarm -- `exc_info=True` is redundant in the `logging` module when you call `exception()`, but possibly not in structlog?

We might need to keep an eye on whether exceptions are still being thrown.

Reply:
I am led to believe it doesnt take `True`, https://docs.python.org/3/library/logging.html#logrecord-objects

> exc_info ([tuple](https://docs.python.org/3/library/stdtypes.html#tuple)[[type](https://docs.python.org/3/library/functions.html#type)[[BaseException](https://docs.python.org/3/library/exceptions.html#BaseException)], [BaseException](https://docs.python.org/3/library/exceptions.html#BaseException), [types.TracebackType](https://docs.python.org/3/library/types.html#types.TracebackType)] | None) – An exception tuple with the current exception information, as returned by [sys.exc_info()](https://docs.python.org/3/library/sys.html#sys.exc_info), or None if no exception information is available.